4.0/start.sh

```diff
@@ -6,22 +6,35 @@ stopServices() {
 }
 trap stopServices TERM
 
+# Needed to create default .env file before calling /app/config.sh if /nominatim is a persistent volume
+if [ ! -f ${PROJECT_DIR}/.env ]; then
+  cat << EOT > ${PROJECT_DIR}/.env 
```

Comment:
If we touch .env we also need to touch this, which is error-prone. Can you put a copy of the .env file inside the container and then copy it into ${PROJECT_DIR} if necessary?